r2/r2/public/static/js/reddit.js

```diff
@@ -1229,6 +1229,17 @@ $(function() {
             $("#moresearchinfo").slideUp();
             event.preventDefault();
         });
+
+        /* Add a scheme to URL input fields if not already present */
+        $("input[type=url]").blur(function() {
```

Comment:
Initially I thought the same, but after checking it looked like this and the corresponding one in [newlink.compact](https://github.com/reddit/reddit/blob/77133cf7ee424267e608ee904d3df5062a435294/r2/r2/templates/newlink.compact#L70) are the only input boxes that uses the `url` type presently: `grep -rn 'type="url"' r2/ --exclude \*css` i.e. there shouldn't be anything for it to potentially break - it would impact future url inputs added (though presumably if it made sense here it would make sense in future ones)